package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-react-toolbox",
-  "version": "0.2.20",
+  "version": "1.0.0",
```

Comment:
Should I do that in this PR if we want to release major?

Reply:
yes